### PR TITLE
fix(sort alphabetically) sorts alphabetically regardless of if something is capitalized or not

### PR DESCRIPTION
**Pull request by thiagocarvalhodev**
### Overview
Today our sort gives preference to file names with upper case. See that TEST FILEs are above the download_progress... file.
We are setting to lower case all names on files and folder before doing the comparison.
image

### CODE REVIEW

File: lib/models/daos/drive_dao/drive_dao.dart [hunk 1]
```diff
@@ -319,8 +319,10 @@ class DriveDao extends DatabaseAccessor<Database> with _$DriveDaoMixin {
       /// it requires triggers, regex spliiting names and creating index fields
       /// and ordering by that index plus interfacing that with moor
       if (orderBy == DriveOrder.name) {
-        subfolders.sort((a, b) => compareNatural(a.name, b.name));
-        files.sort((a, b) => compareNatural(a.name, b.name));
+        subfolders.sort((a, b) =>
+            compareNatural(a.name.toLowerCase(), b.name.toLowerCase()));
```
Comment (arielmelendez) [inline lib/models/daos/drive_dao/drive_dao.dart:323]: Would it be harmful to simply update compareNatural to do the toLowerCase() calls?

Comment (thiagocarvalhodev) [reply lib/models/daos/drive_dao/drive_dao.dart:323]: The `compareNatural` method is from Dart's `collection` library. But we can create another function abstracting the lowerCase operations

**Review by arielmelendez — APPROVED** [review]
Implementation is fine. Will re-approve if you make an update wrapping compareNatural.

**Review by matibat — APPROVED** [review]
LGTM

File: lib/models/daos/drive_dao/drive_dao.dart (hunk 1 above)
Comment (thiagocarvalhodev) [reply lib/models/daos/drive_dao/drive_dao.dart]: https://github.com/ardriveapp/ardrive-web/pull/607/commits/8c193533ea0460f141e006d40a038cf47f703094

File: test/utils/compare_alphabetically_and_natural_test.dart [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    test('should return 1 when 100 is after 99', () {
+      expect(compareAlphabeticallyAndNatural('100', '99'), 1);
+    });
+
+    test('should return -1 when 10a is before 99a', () {
+      expect(compareAlphabeticallyAndNatural('10a', '99a'), -1);
+    });
+
+    test('should return -1 when 1a1 is before 9a9', () {
+      expect(compareAlphabeticallyAndNatural('10a', '99a'), -1);
```
Comment (arielmelendez) [inline test/utils/compare_alphabetically_and_natural_test.dart:25]: Test name doesn't match test value

**Review by arielmelendez — DISMISSED** [review]
LGTM. One minor issue with a test name.

File: test/utils/compare_alphabetically_and_natural_test.dart (hunk 2 above)
Comment (thiagocarvalhodev) [reply test/utils/compare_alphabetically_and_natural_test.dart]: Oh! Good catch! Fixed: https://github.com/ardriveapp/ardrive-web/pull/607/commits/13b0320b1daffdd90cf9e86842bea585b50bba41

**Review by arielmelendez — APPROVED** [review]

**Review by matibat — APPROVED** [review]
Nice!

**Pull request merged**